s3/error/s3errors.go

```diff
@@ -650,6 +651,11 @@ var ErrorCodeResponse = map[S3ErrorCode]S3ErrorStruct{
 		Description:    "The LC configuration specified in the request is invalid.",
 		HttpStatusCode: http.StatusBadRequest,
 	},
+	ErrInvalidVersion: {
+		AwsErrorCode:   "NoSuchBucketVersion",
+		Description:    "The Versioning configuration specified in the request is invalid.",
+		HttpStatusCode: http.StatusBadRequest,
+	},
```

Comment:
It seems that the "ErrInvalidVersioning" defined in the file is confused  with the "ErrInvalidVersion",   and the AwsErrorCode for "ErrInvalidVersion" is not in https://docs.aws.amazon.com/AmazonS3/latest/API/ErrorResponses.html

Reply:
Let me check that

Reply:
Updated